src/crs.jl

```diff
@@ -137,3 +137,53 @@ Base.convert(::Type{CRS}, crs::GFT.CoordinateReferenceSystemFormat) = CRS(crs)
 
 # Maybe enable later, based on https://github.com/JuliaGeo/GeoFormatTypes.jl/issues/21
 # Base.convert(T::Type{<:GFT.CoordinateReferenceSystemFormat}, crs::GFT.CoordinateReferenceSystemFormat) = T(CRS(crs))
+
+
+
+
+"""
+    identify(obj; authority = nothing, options = nothing)
+
+Returns the a list of matching reference CRS or nothing, and the confidence values (0-100) 
+or nothing. This is a high level interface to proj_identify.
```

Comment:
This reads like it could return `nothing`, but that is not the case right? If there is no match they are just empty lists or not?

We can probably leave out the sentence about high level interface, since anything outside of libproj.jl basically is that.

Reply:
empty list now returned, remove mention of high level interface